src/Experimental/Edit.NET/ViewModels/EditSettingsViewModel.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel.DataAnnotations;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Edit.NET.DataModels;
+using TextMateSharp.Grammars;
+
+namespace Edit.NET.ViewModels
+{
+    public partial class EditSettingsViewModel : ObservableValidator
+    {
+        public ObservableCollection<string> AvailableThemes { get; } = new()
+        {
+            "Modern",
+            "ModernContrast",
+            "TurboVision",
+            "TurboVisionCompatible",
+            "TurboVisionGray",
+            "TurboVisionElegant"
+        };
+
+        public ObservableCollection<string> SyntaxThemes { get; } = new();
+
+        public EditSettingsViewModel(Settings settings)
+        {
+            // Initialize properties from current settings
+            SyntaxThemes = new ObservableCollection<string>(Enum.GetNames<ThemeName>());
+            Theme = settings.ConsoloniaTheme.ToString();
+            LightVariant = settings.LightVariant;
+            ShowTabs = settings.ShowTabs;
+            ShowSpaces = settings.ShowSpaces;
+            SelectedSyntaxTheme = settings.SyntaxTheme ?? ThemeName.VisualStudioDark.ToString();
+            DefaultExtension = settings.DefaultExtension;
+        }
+
+        public Settings ToSettings()
+        {
+            return new Settings
+            {
+                ConsoloniaTheme = this.Theme,
```

Comment:
Why don't we bind directly to Settings model, why do we duplicate properties at view model level?